modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -52,10 +72,9 @@ spec:
     resources:
       memory:
         enable_memory_locking: true <1>
-        container:
-          # If omitted, the `min` value is equal to the `max` value (requested resources defaults to limits)
-          # min:
-          max: <number><unit> <2>
+      requests:
```

Comment:
`resources.{requests,limits}` is mutually exclusive with `resources.{memory,cpu,container}`. If requests _and_ limits are provided, memory, cpu, and container, will be ignored.

Memory locking can be enabled with either `statefulset.additionalRedpandaCmdFlags` or some values in `config`. I know the key is `enable_memory_locking` but I'm not sure which stanza it goes into.

Reply:
Our existing doc already mentioned `resources.memory.enable_memory_locking`. Should we keep that as a recommended way of setting memory locking? Or is it better to use `statefulset.additionalRedpandaCmdFlags`?

Reply:
Do we have any baremetal docs that mention memory locking? If so, I'd vote to align with those be it the CLI flag or the rpk config.

If not, I guess it's dealers choice? I don't like `resources.memory.enable_memory_locking` because it's specific to kubernetes. I like to keep interfaces as consistent as possible so transitioning from baremetal to k8s or visa versa will lead to users thinking: "`<blank>` is a CLI flag/RPK Config. Okay, how do I set that in Kubernetes/Baremetal" rather than needing to have a bespoke answer for every piece of configuration.

Reply:
Updated to use `additionalRedpandaCmdFlags`